src/odoc/etc/odoc.css

```diff
@@ -351,6 +351,22 @@ h4 {
   font-size: 1.12em;
 }
 
+/* Comment delimiters, hidden but accessible to screen readers and 
+   selected for copy/pasting */
+
+/* Taken from bootstrap */
+/* See also https://stackoverflow.com/a/27769435/4220738 */
+.comment-delim {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
```

Comment:
Just FTR I personally dislike cargo-culting CSS, that usually leads to pains down the road :-) But based on @Drup [understanding](https://github.com/ocaml/odoc/pull/626#issuecomment-793969920) it seems that in `odig`'s stylesheets you can boil that down to:

```css
.spec .def-doc .comment-delim /* make them invisible yet copy-pastable */
{ position: absolute; width: 1px; height: 1px; overflow: hidden; }
```
(Tested in all browsers major browsers)

Reply:
Yeah, I also think we can shorten it, but there is a "belt and braces": we make it invisible *and* put it somewhere else. :)